src/main.ts

```diff
@@ -37,16 +39,49 @@ export class Minifier {
 
   // Recursively visits every child node, emitting text of the sourcefile that is not a part of
   // a child node.
-  visit(node: ts.Node) {
+  visit(node: ts.Node, typeChecker?: ts.TypeChecker) {
     switch (node.kind) {
-      case ts.SyntaxKind.PropertyAccessExpression: {
-        let output = '';
-        let propAccessExp = <ts.PropertyAccessExpression>node;
-        output += this.visit(propAccessExp.expression);
-        output += '.';
-        // Adds '$mangled' to an identifier to ensure that the identifier is being altered.
-        output += propAccessExp.name.text + '$mangled';
-        return output;
+      case ts.SyntaxKind.Identifier: {
```

Comment:
Talked offline about this: I wanted to avoid having to do string splicing. 

Possible solution is to create a method that does what the code that's currently under the `default` case does, and move these `if`s to `case` statements.